docs/rules/no-document-during-ssr.md

```diff
@@ -0,0 +1,34 @@
+# Disallow references to document during SSR (`lwc/no-document-during-ssr`)
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
+
+Browser APIs must not be accessed when SSR is being done. This rule prevents usage of `document` in `connectedCallback`
+(and in methods called from `conenctedCallback` or anywhere when SSR is being done).
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    connectedCallback() {
+        document.foo = true;
+    }
+}
+```
+
+Examples of **correct** code for this rule:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    renderedCallback() {
+        document.foo = true;
```

Comment:
This is not entirely correct. The `renderedCallback` is invoked every time the component is rendered unlike the `connectedCallback` that is only invoked when the component is connected.

Reply:
Can you expound further? Correctness here roughly translates to "you can do this without it being a problem". If someone wants to set  `document.foo` to `true` on every render, that is perhaps stupid but is not broken.

Would it help to massage the language, use a term other than "correct"? Permitted?